tests/playbooks/roles/install-devstack/defaults/main.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 user: "stack"
 workdir: "/home/{{ user }}/devstack"
-branch: "stable/2023.2"
+branch: "stable/2024.2"
```

Comment:
is the version of gophercloud (we need a newer version to utilize cinder's v3 API - https://github.com/kubernetes/cloud-provider-openstack/issues/2884) preventing us from bumping this to 2025.1 ("Epoxy") right away?

Reply:
Yes, exactly. We need to bump that and I don't want to do so here, since it'll make this PR more complicated than it would be otherwise.